### PR TITLE
Update doc and config to Cloudstate instead of CloudState

### PR DESCRIPTION
**Pull request by michaelpnash**

### CODE REVIEW

File: docs/src/main/paradox/user/lang/java/eventsourced.md [hunk 1]
```diff
@@ -104,6 +104,6 @@ In the example below, we show a shopping cart that also has a checkout command.
 
 ## Registering the entity
 
-Once you've created your entity, you can register it with the @javadoc[`CloudState`](io.cloudstate.javasupport.CloudState) server, by invoking the `registerEventSourcedEntity` method. In addition to passing your entity class and service descriptor, you also need to pass any descriptors that you use for persisting events, for example, the `domain.proto` descriptor.
+Once you've created your entity, you can register it with the @javadoc[`Cloudstate`](io.cloudstate.javasupport.Cloudstate) server, by invoking the `registerEventSourcedEntity` method. In addition to passing your entity class and service descriptor, you also need to pass any descriptors that you use for persisting events, for example, the `domain.proto` descriptor.
```
Comment (jboner) [inline docs/src/main/paradox/user/lang/java/eventsourced.md]: We have not renamed classes yet, right?

Comment (michaelpnash) [reply docs/src/main/paradox/user/lang/java/eventsourced.md]: Good point - my search/replace got carried away, will fix

Comment (viktorklang) [reply docs/src/main/paradox/user/lang/java/eventsourced.md]: @michaelpnash Let me know when you are ready for this to be merged.

**Comment by viktorklang** [comment]
@michaelpnash Ping https://github.com/cloudstateio/cloudstate/pull/131#discussion_r337147365 (this PR also needs rebasing)

**Comment by michaelpnash** [comment]
@viktorklang Ok, done... I think :-)

File: README.md [hunk 2]
```diff
@@ -413,7 +414,7 @@ Next we'll take a look at the Akka proxy, which can be found in [`proxy/core`](p
 
 * [`Serve.scala`](proxy/core/src/main/scala/io/cloudstate/proxy/Serve.scala) - This provides the dynamically implemented gRPC interface as specified by the user function. Requests are forwarded as commands to the cluster sharded persistent entities.
 * [`EventSourcedEntity.scala`](proxy/core/src/main/scala/io/cloudstate/proxy/eventsourced/EventSourcedEntity.scala) - This is an Akka persistent actor that talks to the user function via the event sourcing gRPC protocol.
-* [`CloudStateProxyMain.scala`](proxy/core/src/main/scala/io/cloudstate/proxy/CloudStateProxyMain.scala) - This pulls everything together, starting the Akka gRPC server, cluster sharding, and persistence.
+* [`CloudstateProxyMain.scala`](proxy/core/src/main/scala/io/cloudstate/proxy/CloudstateProxyMain.scala) - This pulls everything together, starting the Akka gRPC server, cluster sharding, and persistence.
```
Comment (viktorklang) [inline README.md]: @michaelpnash  Can't rename classes in name only, you need to rename the class as well.

**Pull request merged**